src/main/java/io/vinyldns/java/model/batch/CreateBatchRequest.java

```diff
@@ -57,6 +64,39 @@ public void setOwnerGroupId(String ownerGroupId) {
     this.ownerGroupId = ownerGroupId;
   }
 
+  public Boolean getAllowManualReview() {
+    return allowManualReview;
+  }
+
+  public void setAllowManualReview(Boolean allowManualReview) {
+    this.allowManualReview = allowManualReview;
+  }
+
+  /** Returns String representation of timestamp in ISO 8601 format */
+  public String getScheduledTime() {
```

Comment:
Oh this isn't swell is it.  I presume this is consistent?  This really should be an Instant in UTC

Reply:
Yeah, that was a bad move; I overhauled this so it should be good now.